homework2/pizza_order.html

```diff
@@ -0,0 +1,89 @@
+<!DOCTYPE html>
+<html lang="en">
+
+<head>
+    <title>Pizza Order</title>
+</head>
+
+<body>
+    <form method="post">
+        <h3>Pizza Order</h3>
+        <label for="name">Your name:</label>
+        <input type="text" id="name" name="name" minlength="3" maxlength="10" required>
+
+        <br><br>
+
+        <label for="emailInput">Your email:</label>
+        <input type="text" id="emailInput" name="email" required>
+
+        <br><br>
+
+        <label for="quantity">Quantity:</label>
+        <select name="quantity">
```

Comment:
aici trebuia sa legi select de label prin id

```suggestion
        <select name="quantity" id="quantity">
```